docs/intelligentapps/finetune.md

```diff
@@ -0,0 +1,278 @@
+---
+Order: 5
+Area: intelligentapps
+TOCTitle: Finetune
+ContentId:
+PageTitle: Finetune AI Models
+DateApproved:
+MetaDescription: Use custom dataset to finetune a generative AI model in the Azure cloud or locally with GPUs. Deploy the finetuned model to the Azure cloud or download adapter file.
+MetaSocialImage:
+---
+
+# Finetune models
+
```

Comment:
Add an introduction about what finetuning is and what this article is about. This article is very long and we can't expect readers to have to go through the entire article to know if this is relevant for them.